tests/test_nmt.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/python
+import numpy as np
+import os
+import pymaster as nmt
```

Comment:
I added these tests to .github/workflows for continuous integration. pymaster installation is failing in those tests. Please check. We should make sure continuous integration works before merging.

Reply:
Fixed. There were two problems:
* NaMaster does not declare its dependencies in the `setup.py` and does not wait for them to be installed before compilation (https://github.com/LSSTDESC/NaMaster/issues/138). This is solved by installing the packages in `requirements.txt` with a loop and `pip install -U`.
* NaMaster needs a few non-python packages to be installed. 

I also added `wheel` to the continuous integration to avoid recompiling camb, ccl and pymaster, if they're found in the cache.

Reply:
Great! Thank you for the fixes.